cmd/awscollector/main_others.go

```diff
@@ -20,15 +20,14 @@
 package main
 
 import (
+	"go.opentelemetry.io/collector/otelcol"
```

Comment:
nit: Same issue as above with the ordering of imports.

Reply:
Thanks. Updated